### ISSUE TITLE
Support custom trivy image instead of aquasec/trivy

### ISSUE THREAD

**Issue by BedivereZero** [aquasecurity/trivy-azure-pipelines-task#20]
In Azure DevOps for privatized deployments, we pull images from `registry.my-company.com/third-party/trivy` instead of `aquasec/trivy` because of network isolation.

**Comment by abdurrahman-osman** [comment]
Is there any update on this issue?

**Comment by a-poluyanov** [comment]
We need this update to be able to scan in a closed environment.

**Comment by SSchwaiger** [comment]
Same here. Would be delighted to have this line confiugurable: https://github.com/aquasecurity/trivy-azure-pipelines-task/blob/17eb1a938878a39c491d37967274a77551fb0b40/trivy-task/index.ts#L132

**Comment by sellilo** [comment]
> Same here. Would be delighted to have this line confiugurable:
> 
> https://github.com/aquasecurity/trivy-azure-pipelines-task/blob/17eb1a938878a39c491d37967274a77551fb0b40/trivy-task/index.ts#L132

Yeah, would just be nice to have an optional config parameter "REGISTRY" with a String that is concatenated with the "aquasec/trivy + Version".

Similar to https://github.com/aquasecurity/trivy-azure-pipelines-task/issues/78 this could help to use trivy in protected environments (via a registry-proxy in docker mode) and save a lot of download time a well (for "binary" mode).

**Comment by a-poluyanov** [comment]
> > Same here. Would be delighted to have this line confiugurable:
> > https://github.com/aquasecurity/trivy-azure-pipelines-task/blob/17eb1a938878a39c491d37967274a77551fb0b40/trivy-task/index.ts#L132
> 
> Yeah, would just be nice to have an optional config parameter "REGISTRY" with a String that is concatenated with the "aquasec/trivy + Version".
> 
> Similar to #78 this could help to use trivy in protected environments (via a registry-proxy in docker mode) and save a lot of download time a well (for "binary" mode).

Apparently they don't want to make any changes. Maybe if you have time, you can make changes in your fork and make a pull request.
